lighthouse.ts

```diff
@@ -14,7 +14,7 @@ const runReport = async (url, outputFormat, maxWait) => {
     `--emulated-form-factor=mobile`,
     `--only-categories=performance`,
     `--chrome-flags="--headless"`,
-    `--max-wait-for-load=${maxWait || 45000}`,
```

Comment:
The thinking here was that it might be helpful for testing purposes to be able to override the timeout, presumably to cause it immediately and speed up the test. I can't remember if it's being used anywhere, yet...

To be clear, it was a workaround to compensate for bad architecture. The right way to do this is to actually mock the lighthouse logic in such a way that live never actually happen during testing. But I haven't gotten to figuring that out.

The bottom line is that I don't care if we remove this but if we do, we should have another strategy to replace what it was supposed to do.

Reply:
I took it out because the one place we are calling this function, we aren't passing in the parameter. I think it is easy enough to add it back if we end up using it in the future.

Reply:
Yea I'm remembering that I think the slow/unstable live network request problem is kind of temporarily "solved" by lack of test coverage for that code. 🤪